### PR TITLE
Add ECS Health Metrics Script

### PR DESCRIPTION
**Pull request by MinhThieu145**
Add ECS health metrics. Add in one script `monitor_ecs_metrics`

### CODE REVIEW

File: scripts/monitoring/monitor_ecs_metrics.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    "MEMCACHED_LOCATION": os.environ.get("MEMCACHED_LOCATION", None),
+    "RDS_DB_NAME": settings.DATABASES["default"]["NAME"],
+    "RDS_HOSTNAME": settings.DATABASES["default"]["HOST"],
+    "RDS_PASSWORD": settings.DATABASES["default"]["PASSWORD"],
+    "RDS_USERNAME": settings.DATABASES["default"]["USER"],
+    "RDS_PORT": settings.DATABASES["default"]["PORT"],
+    "SECRET_KEY": settings.SECRET_KEY,
+    "SENTRY_URL": os.environ.get("SENTRY_URL"),
+    "STATSD_ENDPOINT": os.environ.get("STATSD_ENDPOINT"),
+    "STATSD_PORT": os.environ.get("STATSD_PORT"),
```
Comment (gchhablani) [inline scripts/monitoring/monitor_ecs_metrics.py:69]: Are all of these used?

File: scripts/monitoring/monitor_ecs_metrics.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    "AUTH_TOKEN",
+)
+evalai_endpoint = os.environ.get("API_HOST_URL")
+authorization_header = {
+    "Authorization": "Bearer {}".format(os.environ.get("AUTH_TOKEN"))
+}
+
+DJANGO_SETTINGS_MODULE = os.environ.get("DJANGO_SETTINGS_MODULE")
+ENV = DJANGO_SETTINGS_MODULE.split(".")[-1]
+EVALAI_DNS = os.environ.get("SERVICE_DNS")
```
Comment (gchhablani) [inline scripts/monitoring/monitor_ecs_metrics.py:19]: Do we need these variables?

File: scripts/monitoring/monitor_ecs_metrics.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        StartTime=start_time,
+        EndTime=end_time,
+        Period=period_seconds,
+        Statistics=["Average", "Maximum", "Minimum"],
+    )
+
+    return response["Datapoints"]
+
+
+def get_storage_metrics_for_challenge(challenge, cluster_name=COMMON_SETTINGS_DICT["CLUSTER"], range_days=1, period_seconds=300):
```
Comment (gchhablani) [inline scripts/monitoring/monitor_ecs_metrics.py:152]: Use `evalai-production-cluster` as the default cluster name.

**Review by gchhablani — COMMENTED** [review]
Added some comments.

File: scripts/monitoring/monitor_ecs_metrics.py (hunk 1 above)
Comment (MinhThieu145) [reply scripts/monitoring/monitor_ecs_metrics.py:69]: No, only the CLUSTER one. I'll remove the unused

File: scripts/monitoring/monitor_ecs_metrics.py (hunk 2 above)
Comment (MinhThieu145) [reply scripts/monitoring/monitor_ecs_metrics.py:19]: No. I'll remove them

File: scripts/monitoring/monitor_ecs_metrics.py (hunk 3 above)
Comment (MinhThieu145) [reply scripts/monitoring/monitor_ecs_metrics.py:152]: For sure.

**Pull request closed without merging**